README.md

```diff
@@ -166,11 +166,16 @@ For nvim-cmp, it is also possible to follow the [example recommendation](https:/
 
 ## Add Snippets
 
-- **Vscode-like**: For using snippets from a plugin (eg. [rafamadriz/friendly-snippets](https://github.com/rafamadriz/friendly-snippets)) install it and add
+- **Vscode-like**: To use existing vs-code style snippets from a plugin (eg. [rafamadriz/friendly-snippets](https://github.com/rafamadriz/friendly-snippets)) simply install the friendly-snippets plugin and then add
     ```lua
     require("luasnip.loaders.from_vscode").load()
     ```
-	somewhere in your config.  
+	somewhere in your nvim config. LuaSnip will then automatically load the snippets contained in the friendly-snippets plugin.
+  You can also easily **load your own custom vscode style snippets** by stating the path to the custom snippets folder within the load function:
+    ```lua
+    -- in your nvim folder
```

Comment:
What do you think of "will load snippets from `path/of/your/nvim/config/my-cool-snippets`"?

Reply:
yeh that's better, covers all bases then.